docs/packaging-your-sql-project/etl-eda-pipeline-with-ploomber.md

```diff
@@ -0,0 +1,187 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.6
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Expanding our ETL pipeline into an ETL-EDA pipeline
+
+In the last section, we learned how to combine our ETL processes into a single Python script. We also learned how to run this script from the command line. In this section, we will learn how to expand our ETL pipeline into an ETL-EDA pipeline. We will learn how to incorporate exploratory data analysis (EDA) into our pipeline, and how to save the results of our EDA into a database. In this tutorial, we will introduce a tool that will help us package `.py` scripts along with `.ipynb` notebooks into a single pipeline: Ploomber. 
```

Comment:
"In this tutorial, we will introduce a tool that will help us package `.py` scripts along with `.ipynb` notebooks into a single pipeline: Ploomber." 

recommended change to:

In this tutorial, we will introduce [Ploomber](https://docs.ploomber.io/en/latest/), a tool that will help us package `.py` scripts along with `.ipynb` notebooks into a single pipeline.

Reply:
Fixed